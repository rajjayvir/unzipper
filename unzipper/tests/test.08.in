0x36
0x22 0x50 0x00
0x3b 0x50 0x80
0x0a 0x41 0x00
0x77 0x82 0x00
0x3e 0x82 0x10
0x2f 0x92 0x20
0x38 0xa2 0x28
0x37 0xa2 0x2c
0x7a 0x82 0x30
0x6c 0x62 0x40
0x66 0x62 0x80
0x2c 0x62 0xc0
0x69 0x43 0x00
0x74 0x44 0x00
0x72 0x55 0x00
0x2b 0x85 0x80
0x31 0x95 0x90
0x26 0x95 0x98
0x68 0x75 0xa0
0x64 0x65 0xc0
0x29 0x56 0x00
0x28 0x56 0x80
0x6e 0x57 0x00
0x30 0x77 0x80
0x23 0x87 0xa0
0x25 0x87 0xb0
0x3c 0x87 0xc0
0x2e 0x87 0xd0
0x78 0x97 0xe0
0x5d 0x97 0xe8
0x55 0xa7 0xf0
0x4e 0xa7 0xf4
0x2a 0xa7 0xf8
0x6b 0xa7 0xfc
0x73 0x58 0x00
0x4c 0x98 0x80
0x32 0x98 0x88
0x5c 0x98 0x90
0x5b 0x98 0x98
0x3d 0x78 0xa0
0x6f 0x68 0xc0
0x61 0x59 0x00
0x70 0x59 0x80
0x63 0x6a 0x00
0x62 0x6a 0x40
0x6d 0x5a 0x80
0x65 0x5b 0x00
0x7d 0x7b 0x80
0x7b 0x7b 0xa0
0x5f 0x8b 0xc0
0x21 0x8b 0xd0
0x75 0x8b 0xe0
0x67 0x8b 0xf0
0x20 0x2c 0x00
0x00 0x00 0x00 0x00
