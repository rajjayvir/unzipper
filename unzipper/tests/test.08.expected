0x22 : 5 00000
0x3b : 5 00001
0x0a : 4 0001
0x77 : 8 00100000
0x3e : 8 00100001
0x2f : 9 001000100
0x38 : 10 0010001010
0x37 : 10 0010001011
0x7a : 8 00100011
0x6c : 6 001001
0x66 : 6 001010
0x2c : 6 001011
0x69 : 4 0011
0x74 : 4 0100
0x72 : 5 01010
0x2b : 8 01011000
0x31 : 9 010110010
0x26 : 9 010110011
0x68 : 7 0101101
0x64 : 6 010111
0x29 : 5 01100
0x28 : 5 01101
0x6e : 5 01110
0x30 : 7 0111100
0x23 : 8 01111010
0x25 : 8 01111011
0x3c : 8 01111100
0x2e : 8 01111101
0x78 : 9 011111100
0x5d : 9 011111101
0x55 : 10 0111111100
0x4e : 10 0111111101
0x2a : 10 0111111110
0x6b : 10 0111111111
0x73 : 5 10000
0x4c : 9 100010000
0x32 : 9 100010001
0x5c : 9 100010010
0x5b : 9 100010011
0x3d : 7 1000101
0x6f : 6 100011
0x61 : 5 10010
0x70 : 5 10011
0x63 : 6 101000
0x62 : 6 101001
0x6d : 5 10101
0x65 : 5 10110
0x7d : 7 1011100
0x7b : 7 1011101
0x5f : 8 10111100
0x21 : 8 10111101
0x75 : 8 10111110
0x67 : 8 10111111
0x20 : 2 11
